File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/Adm107UFwkUserIdInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    public class Adm107UFwkUserIdInfo
    {
        private String _userId;
        private String _userNm;
        private String _groupUser;

        public String UserId
        {
            get { return this._userId; }
            set { this._userId = value; }
        }

        public String UserNm
        {
            get { return this._userNm; }
            set { this._userNm = value; }
        }

        public String GroupUser
        {
            get { return this._groupUser; }
            set { this._groupUser = value; }
        }

        public Adm107UFwkUserIdInfo() { }

        public Adm107UFwkUserIdInfo(String userId, String userNm, String groupUser)
        {
            this._userId = userId;
            this._userNm = userNm;
            this._groupUser = groupUser;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/Adm107UFwkUserIdInfo.java
package nta.med.data.model.ihis.adma;

public class Adm107UFwkUserIdInfo {
	private String userId ;
	private String userNm ;
	private String groupUser ;
	public Adm107UFwkUserIdInfo(String userId, String userNm, String groupUser) {
		super();
		this.userId = userId;
		this.userNm = userNm;
		this.groupUser = groupUser;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public String getUserNm() {
		return userNm;
	}
	public void setUserNm(String userNm) {
		this.userNm = userNm;
	}
	public String getGroupUser() {
		return groupUser;
	}
	public void setGroupUser(String groupUser) {
		this.groupUser = groupUser;
	}
	
}
